models/scheduler.go

```diff
@@ -248,7 +248,13 @@ func (c *Scheduler) NextMajorVersion() {
 	c.Version = fmt.Sprintf("v%d.0", major+1)
 }
 
-// NextMinorVersion increments the major version
+// PreviousMajorVersion decrements the major version
```

Comment:
Why would you need to decrement the major version?

Reply:
I don't need anymore, gonna take it off. I was using it in `getOperationRollingProgress`